.github/workflows/ibis-backends.yml

```diff
@@ -827,6 +828,10 @@ jobs:
       - name: install ibis
         run: poetry install --without dev --without docs --extras flink
 
+      # TODO(deepyaman): Remove step once Ibis and Flink are compatible.
+      - name: install pyflink
+        run: pip install apache-flink
```

Comment:
This looks like it's installing `apache-flink` for every backend. You should be able to install it only for the `flink` backend by using `if: ${{ matrix... }}` on the job step.

Reply:
Same thing: foiled by the GitHub UI code folding.